packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Providers/MrwSerializationTypeDefinition.cs

```diff
@@ -710,6 +711,19 @@ private List<MethodBodyStatement> BuildDeserializePropertiesStatements(ScopedApi
             Dictionary<JsonValueKind, List<MethodBodyStatement>> additionalPropsValueKindBodyStatements = [];
             var parameters = SerializationConstructor.Signature.Parameters;
 
+            // Parse the custom serialization attributes
+            IEnumerable<AttributeData> serializationAttributes = _model.CustomCodeView?.GetAttributes()
+                .Where(a => a.AttributeClass?.Name == CodeGenAttributes.CodeGenSerializationAttributeName) ?? [];
+            var baseModelProvider = _model.BaseModelProvider;
+
+            while (baseModelProvider != null)
+            {
+                serializationAttributes = serializationAttributes
+                    .Concat(baseModelProvider.CustomCodeView?.GetAttributes()
+                    .Where(a => a.AttributeClass?.Name == CodeGenAttributes.CodeGenSerializationAttributeName) ?? []);
```

Comment:
linq method could be very low efficient because every time you call Concat it creates a new IEnumerable instance, could we change this to a List and call AddRange?

Reply:
sure, thanks for the tip.